get-subseqs

```diff
@@ -1,3 +1,74 @@
-#!/bin/bash
+#!/usr/bin/python3
 
-python3 src/get-subseqs.py $@
+import sys
+import numpy as np
+
+def get_subseqs():
+
+    try:
+        inFile_1 = sys.argv[1]
+        inFile_2 = sys.argv[2]
+
+        with open(inFile_1,'r') as f1:
+            f1_lines = f1.readlines()
+        with open(inFile_2,'r') as f2:
+            f2_lines = f2.readlines()
+
+        record_list = []
+        header = ''
+        sequence = ''
+        interval = []
+        for line in f1_lines:
+            line = line.strip()
+            if line.startswith('>'):
+                if header != "":
+                    record_list.append([header.strip(), sequence.strip()])
+                    sequence = ""
+                header = line[1:]
+            else:
+                sequence = sequence + line
+        record_list.append([header.strip(), sequence.strip()])
```

Comment:
here again you need to concatenate the lines in `sequence` before you strip: `''.join(sequence).strip()`